### ISSUE TITLE
Add new podcasts to iPhone App

### ISSUE THREAD

**Issue by vsinghkpcc** [SCPR/KPCC-iPhone#57]
AC

1) Gather New Podcast avatars and hand them off to Ben H.
2) Update call we make to program endpoints to include "online" and "on-air"

**Comment by sdillingham** [comment]
Requested portrait photo for 5 Every Week and The Wheel Thing from Collin. Blocked on this ticket until I get creative from him

**Comment by sdillingham** [comment]
Checked in With Collin. Still no assets from 5 Every Week, and Welcome to Vinci has ended. I'd suggest moving this into the backlog for now.

**Comment by sdillingham** [comment]
Since these are now being included in @ewr's latest build, I think we should update the AC for this one and consider it in Review. The avatars and art can come later in a separate ticket. @vsinghkpcc - thoughts?

**Comment by vsinghkpcc** [comment]
@sdillingham Agreed!

**Comment by ewr** [comment]
This is in the app version that is currently awaiting review.
